app/src/main/java/com/cortado/minionify/mode/AppMode.java

```diff
@@ -0,0 +1,16 @@
+package com.cortado.minionify.mode;
+
+public enum AppMode {
+    MINION("Minion"),
+    MANAGER("Manager");
+
+    private String mValue;
```

Comment:
why not integer? no real use for this string
also may be final

Reply:
Because of AppMode.valueOf(String name):
AppMode.valueOf("MINION") == AppMode.MINION
AppMode.valueOf(1) != AppMode.MINION

Reply:
add a method "fromValue"